src/components/breadcrumb-item/breadcrumb-item.ts

```diff
@@ -34,13 +34,19 @@ export default class SlBreadcrumbItem extends LitElement {
   /** Tells the browser where to open the link. Only used when `href` is set. */
   @property() target: '_blank' | '_parent' | '_self' | '_top';
 
+  /** Optionally allows the user to determine how the link should talk to the browser.
+   * ref: https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types
+   */
+  @property() rel: string | undefined;
+
   handleSlotChange() {
     this.hasPrefix = hasSlot(this, 'prefix');
     this.hasSuffix = hasSlot(this, 'suffix');
   }
 
   render() {
     const isLink = this.href ? true : false;
+    const rel = this.rel ? this.rel : 'noreferrer noopener';
```

Comment:
I'm curious if there's a reason you're setting the default here instead of on line 40. I typically set the default in the prop's definition so the CEM analyzer picks it up and it gets shown in the docs.

Reply:
Because we wouldn't want it to be more succinct would we 🙄 /sarcasm 🤦🏼